docs/setup/asb/systemd/start-asb.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Run the automated swap backend
+After=start-monero-wallet-rpc.service
```

Comment:
Probably want a `Requires` as well to enforce that this doesn't start if the wallet fails.